src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -425,13 +436,15 @@ public void simulationPeriodic() {
     }
 
     // Wrist Sim Stuff
-    m_wristMotorSim.setInput(wristMotor.get() * RobotController.getBatteryVoltage());
+    m_wristMotorSim.setInput(wristMotor.getAppliedOutput() * RobotController.getBatteryVoltage());
```

Comment:
I think getAppliedOutput() returns the voltage applied, so multiplying by the battery voltage sets the input to 12x what it should be